### ISSUE TITLE
Arctic/vc1 recipe not working

### ISSUE THREAD

**Issue by shehrum** [espnet/espnet#2621]
I am trying to run the arctic/vc1 recipe, however, I run into the following error:

> download_from_google_drive_2.sh: command not found

I think this is a missing file. 

**Environment information**
```
PyTorch version: 1.4.0
Is debug build: No
CUDA used to build PyTorch: 10.0

OS: Ubuntu 18.04.5 LTS
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: version 3.18.2

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration: GPU 0: Tesla K80
Nvidia driver version: 450.80.02
cuDNN version: Probably one of the following:
/usr/local/cuda-10.1/targets/x86_64-linux/lib/libcudnn.so.7.6.5
/usr/local/cuda-10.2/targets/x86_64-linux/lib/libcudnn.so.7.6.5
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_adv_train.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8.0.4
/usr/local/cuda-11.0/targets/x86_64-linux/lib/libcudnn_ops_train.so.8.0.4

Versions of relevant libraries:
[pip3] numpy==1.19.2
[pip3] pytorch-lamb==1.0.0
[pip3] pytorch-ranger==0.1.1
[pip3] pytorch-wpe==0.0.0
[pip3] torch==1.4.0
[pip3] torch-complex==0.2.0
[pip3] torch-optimizer==0.0.1a16
[pip3] torchaudio==0.4.0
[pip3] torchvision==0.8.1
[pip3] warpctc-pytorch==0.2.1
[pip3] warprnnt-pytorch==0.1
[conda] mkl                       2020.2                      256  
[conda] pytorch                   1.4.0           py3.7_cuda10.0.130_cudnn7.6.3_0    pytorch
[conda] pytorch-lamb              1.0.0                     dev_0    <develop>
[conda] pytorch-ranger            0.1.1                    pypi_0    pypi
[conda] pytorch-wpe               0.0.0                    pypi_0    pypi
[conda] torch-complex             0.2.0                    pypi_0    pypi
[conda] torch-optimizer           0.0.1a16                 pypi_0    pypi
[conda] torchaudio                0.4.0                    pypi_0    pypi
[conda] torchvision               0.8.1                    pypi_0    pypi
[conda] warpctc-pytorch           0.2.1                    pypi_0    pypi
[conda] warprnnt-pytorch          0.1                       dev_0    <develop>
```

**Task information:**
 - Task: [VC1]
 - Recipe: [ARCTIC]
 - ESPnet1 

**To Reproduce**
1. move to a recipe directory, e.g., `cd egs/arctic/vc1`
2. execute `run.sh` 

**Error logs**
```
stage -1: Data and Pretrained Model Download
--2020-11-03 00:08:32--  http://festvox.org/cmu_arctic/cmu_arctic/packed/cmu_us_clb_arctic-0.95-release.tar.bz2
Resolving festvox.org (festvox.org)... 199.4.150.154
Connecting to festvox.org (festvox.org)|199.4.150.154|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 130040259 (124M) [application/x-bzip2]
Saving to: 'cmu_us_clb_arctic-0.95-release.tar.bz2'

cmu_us_clb_arctic-0.95-release.tar.bz2             100%[================================================================================================================>] 124.02M  22.0MB/s    in 12s     

2020-11-03 00:08:44 (10.7 MB/s) - 'cmu_us_clb_arctic-0.95-release.tar.bz2' saved [130040259/130040259]

Successfully finished download.
--2020-11-03 00:08:58--  http://festvox.org/cmu_arctic/cmu_arctic/packed/cmu_us_slt_arctic-0.95-release.tar.bz2
Resolving festvox.org (festvox.org)... 199.4.150.154
Connecting to festvox.org (festvox.org)|199.4.150.154|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 119914432 (114M) [application/x-bzip2]
Saving to: 'cmu_us_slt_arctic-0.95-release.tar.bz2'

cmu_us_slt_arctic-0.95-release.tar.bz2             100%[================================================================================================================>] 114.36M  22.0MB/s    in 10s     

2020-11-03 00:09:08 (11.3 MB/s) - 'cmu_us_slt_arctic-0.95-release.tar.bz2' saved [119914432/119914432]

Successfully finished download.
local/pretrained_model_download.sh: line 31: download_from_google_drive_2.sh: command not found

```

**Comment by kan-bayashi** [comment]
Could you fix download_from_google_drive_2.sh -> download_from_google_drive.sh?

**Comment by kan-bayashi** [comment]
fixed via #2622
